app/views/documents/show/_scheduling_notice.html.erb

```diff
@@ -0,0 +1,49 @@
+<% datetime = @edition.scheduled_publishing_datetime %>
+<% time = datetime&.strftime("%l:%M%P")&.strip %>
```

Comment:
Do you know what the strip is for here?

Reply:
I think that's a hangover from another view where we'd sometimes get additional spacing.  That doesn't seem to be a problem with this view so I'll get rid of it.